csharp/ql/src/semmle/code/csharp/frameworks/system/xml/XPath.qll

```diff
@@ -21,4 +21,14 @@ module SystemXmlXPath {
   class XPathExpression extends Class {
     XPathExpression() { this.hasName("XPathExpression") }
   }
+
+  /** The `System.Xml.XPath.XPathNavigator` class. */
+  class XPathNavigator extends Class {
```

Comment:
It looks like this class also contains the following methods which accept XPath strings:
 - `Compile`
 - `Evaluate`
 - `Matches`

Reply:
We also look like we are missing:
 * `XPathExpression.Compile`

Reply:
`XPathExpression.Compile` is already covered; I will add the others.